charts/modules/apps/argo-cd/README.md

```diff
@@ -81,6 +81,8 @@ sequenceDiagram
 
 
 
+
```

Comment:
please delete this by running readme generator OR fix the readme generator if there is a fix (max time 10-15 minutes)